backend/db/migrate/20220303132723_create_open_calls.rb

```diff
@@ -0,0 +1,38 @@
+class CreateOpenCalls < ActiveRecord::Migration[7.0]
+  def change
+    create_table :open_calls, id: :uuid do |t|
+      t.belongs_to :investor, foreign_key: {on_delete: :cascade}, type: :uuid, null: false, index: true
+
+      t.text :name_en
+      t.text :name_es
+      t.text :name_pt
+      t.text :slug
+      t.text :description_en
+      t.text :description_es
+      t.text :description_pt
+      t.text :money_distribution_en
+      t.text :money_distribution_es
+      t.text :money_distribution_pt
+      t.text :impact_description_en
+      t.text :impact_description_es
+      t.text :impact_description_pt
+
+      t.boolean :trusted, null: false, default: false
+
+      t.string :ticket_size, null: false
+      t.string :instrument_type, null: false
```

Comment:
:) super simple to break integrity of these records when you start manipulating with TYPES -- but again, I know that you don't except anybody to change TYPES, so you can definitely ignore this ;)

Reply:
All of those are closed lists which will have descriptions etc. that will be pushed to transifex at some point. We need to easy way to push/pull translations for that and Rails locale files seem like a good choice.

Reply:
Thanks @tsubik , I know that you have already mentioned this at email. Will not mentioned this again, don't worry :)